src/App.js

```diff
@@ -35,25 +50,43 @@ function App() {
       <Header />
 
       <SearchBar
+        value={searchValue}
+        waveIndex={waveIndex}
         onSearchChange={searchChangeHandler}
         onSearchSelect={searchSelectHandler}
       />
 
+      <Tabs
+        className="tabs-container"
+        selectedIndex={waveIndex}
+        onSelect={tabChangeHandler}
+      >
+        <TabList>
+          {Array.from({ length: waves }, (_, i) => (
+            <Tab key={i}>Wave {i + 1}</Tab>
```

Comment:
As discussed, it'd be nice if we could have a `waves.json` file which will hold these details (assuming 0-based index waves):

```
{
  "waves": {
    0: {
      "name": "Wave 1",
      "isArchived": false,
    },
    1: {
      "name": "Wave 2",
      "isArchived": false,
    }
  }
}
```
All archived waves will then be collated into an "Archives" tab.

Reply:
Sure. Maybe you can make another PR to modify the Python script to incorporate this and generate the `wave` attribute for the mentors also.